includes/Modules/Sign_In_With_Google/Web_Tag.php

```diff
@@ -197,20 +197,41 @@ protected function render() {
 			 * Mainly used by Gutenberg blocks.
 			 */
 			?>
-		document.querySelectorAll( '.googlesitekit-sign-in-with-google__frontend-output-button' ).forEach( ( siwgButtonDiv ) => {
-			google.accounts.id.renderButton( siwgButtonDiv, <?php echo wp_json_encode( $btn_args ); ?> );
-		});
-	<?php endif; // phpcs:ignore Generic.WhiteSpace.ScopeIndent.Incorrect ?>
-
-	<?php if ( $should_show_one_tap_prompt ) : // phpcs:ignore Generic.WhiteSpace.ScopeIndent.Incorrect ?>
-		google.accounts.id.prompt();
-	<?php endif; // phpcs:ignore Generic.WhiteSpace.ScopeIndent.Incorrect ?>
-
-	<?php if ( ! empty( $this->redirect_to ) ) : // phpcs:ignore Generic.WhiteSpace.ScopeIndent.Incorrect ?>
-		const expires = new Date();
-		expires.setTime( expires.getTime() + <?php echo esc_js( $cookie_expire_time ); ?> );
-		document.cookie = "<?php echo esc_js( Authenticator::COOKIE_REDIRECT_TO ); ?>=<?php echo esc_js( $this->redirect_to ); ?>;expires=" + expires.toUTCString() + ";path=<?php echo esc_js( Authenticator::get_cookie_path() ); ?>";
-	<?php endif; // phpcs:ignore Generic.WhiteSpace.ScopeIndent.Incorrect ?>
+const defaultButtonOptions = <?php echo wp_json_encode( $btn_args ); ?>;
+document.querySelectorAll( '.googlesitekit-sign-in-with-google__frontend-output-button' ).forEach( ( siwgButtonDiv ) =>
+{
+const buttonOptions = Object.assign( {}, defaultButtonOptions );
+const textAttribute = siwgButtonDiv.getAttribute( 'data-googlesitekit-siwg-text' );
+const themeAttribute = siwgButtonDiv.getAttribute( 'data-googlesitekit-siwg-theme' );
+const shapeAttribute = siwgButtonDiv.getAttribute( 'data-googlesitekit-siwg-shape' );
+
+if ( textAttribute ) {
+buttonOptions.text = textAttribute;
+}
+
+if ( themeAttribute ) {
+buttonOptions.theme = themeAttribute;
+}
+
+if ( shapeAttribute ) {
+buttonOptions.shape = shapeAttribute;
+}
+
+google.accounts.id.renderButton( siwgButtonDiv, buttonOptions );
```

Comment:
This should work fine and be more concise:

```suggestion
const buttonOptions = {
	shape: siwgButtonDiv.getAttribute( 'data-googlesitekit-siwg-shape' ) || defaultButtonOptions.shape,
	text: siwgButtonDiv.getAttribute( 'data-googlesitekit-siwg-text' ) || defaultButtonOptions.text,
	theme: siwgButtonDiv.getAttribute( 'data-googlesitekit-siwg-theme' ) || defaultButtonOptions.theme,
};

google.accounts.id.renderButton( siwgButtonDiv, buttonOptions );
```